workflows/shareseq/shareseq_workflow.wdl

```diff
@@ -0,0 +1,420 @@
+                        preemptible = preemptible,
+                        awsMaxRetries = awsMaxRetries,
+                        backend = backend
+                }
+            }
+        }
+
+        if (length(generate_count_config.sample_atac_ids) > 0) {
+            scatter (sample_id in generate_count_config.sample_atac_ids) {
+                call shr.shareseq_reorg as shareseq_reorg_atac {
+                    input:
+                        sample_id = sample_id,
+                        type = 'atac',
+                        input_fastqs_directories = generate_count_config.sample2dir[sample_id],
+                        r1_fastq_pattern = read1_fastq_pattern,
+                        r2_fastq_pattern = read2_fastq_pattern,
+                        index_fastq_pattern = index_fastq_pattern,
+                        output_directory = output_directory_stripped,
+                        shareseq_reorg_version = shareseq_reorg_version,
+                        docker_registry = docker_registry,
+                        zones = zones,
+                        num_cpu = sharseq_reorg_num_cpu,
+                        memory = shareseq_reorg_memory,
+                        disk_space = shareseq_reorg_disk_space,
+                        preemptible = preemptible,
+                        awsMaxRetries = awsMaxRetries,
+                        backend = backend   
+                }    
+                call cm.chromap_mapping as chromap {
+                    input:
+                        chromap_version = chromap_version,
+                        read1_fastq_pattern = '_f*_R1.fastq.gz',
+                        read2_fastq_pattern = '_f*_R2.fastq.gz',
+                        barcode_fastq_pattern = '_f*_I1.fastq.gz',
+                        preset = 'atac',
+                        acronym_file = acronym2gsurl['chromap'],
+                        sample_id = sample_id,
+                        output_directory = output_directory_stripped,
+                        input_fastqs_directories = shareseq_reorg_atac.output_reorg_directory,
+                        genome = generate_count_config.sample2genome[sample_id],
+                        disk_space = chromap_disk_space,
+                        docker_registry = docker_registry_stripped,
+                        zones = zones,
+                        num_cpu = chromap_num_cpu,
+                        memory = chromap_memory,
+                        preemptible = preemptible,
+                        awsMaxRetries = awsMaxRetries,
+                        backend = backend
+                }
+            }
+        }      
+    }
+    output {
+        Array[String]? demuxed_fastqs = shareseq_mkfastq.output_fastqs_directory
+        Array[String]? reorg_gex_fastqs = shareseq_reorg_gex.output_reorg_directory
+        Array[String]? reorg_atac_fastqs = shareseq_reorg_atac.output_reorg_directory 
+        Array[String]? gex_outputs = starsolo_count.output_folder
```

Comment:
Yiming just make another PR (merged), in which starsolo_count.output_folder changes to starsolo_count.output_count_directory. Please make the change accordingly. Thanks!